### PR TITLE
add PIC32 related crates

### PR DESCRIPTION
**Pull request by kiffie**
Add PIC32 related crates, as proposed by @therealprof.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,6 +117,7 @@ In 2018 the Rust community created an embedded working group to help drive adopt
 -   [xargo](https://github.com/japaric/xargo) Rust package manager with support for non-default std libraries — build rust runtime for your own embedded system.
     - xargo is great but it's in maintenance mode, [cargo-xbuild](https://github.com/rust-osdev/cargo-xbuild) is catching up as intended replacement.
 -   [svd2rust](https://github.com/japaric/svd2rust) Generate Rust structs with register mappings from SVD files.
+-   [edc2svd](https://github.com/kiffie/edc2svd) Generate SVD files for PIC32 devies from EDC files. - ![crates.io](https://img.shields.io/crates/v/edc2svd.svg)
```
Comment (jamesmunns) [inline README.md:120]:
```suggestion
-   [edc2svd](https://github.com/kiffie/edc2svd) Generate SVD files for PIC32 devices from EDC files. - ![crates.io](https://img.shields.io/crates/v/edc2svd.svg)
```

**Comment by jamesmunns** [comment]
One quick type, then looks good to me! Thanks for sharing!

**Review by jamesmunns — APPROVED** [review]
Went ahead and applied the spelling fix. Thanks!

bors r+

**Comment by kiffie** [comment]
Thanks for this extremely quick merge. And sorry about the typo.

**Pull request merged**